pom.xml

```diff
@@ -804,6 +804,18 @@
         <scope>compile</scope>
       </dependency>
 
+      <!-- metrics -->
+      <dependency>
+        <groupId>com.spotify.metrics</groupId>
```

Comment:
fyi @stas added a BOM to ffwd so we don't have to have multiple dependencies. just need to release it.

https://github.com/spotify/semantic-metrics/blob/master/semantic-metrics-bom/pom.xml